src/osg/Texture2DMultisample.cpp

```diff
@@ -102,32 +102,29 @@ void Texture2DMultisample::apply(State& state) const
     }
     else if ( (_textureWidth!=0) && (_textureHeight!=0) && (_numSamples!=0) )
     {
-        // no image present, but dimensions at set so lets create the texture
```

Comment:
I think this whole file can go back to the 3.6 version now that selectSizedInternalFormat is going back into Texture.cpp